readthedocs/rtd_tests/tests/test_config_integration.py

```diff
@@ -75,38 +75,45 @@ def setUp(self):
 
     @mock.patch('readthedocs.doc_builder.config.load_config')
     def test_python_supported_versions_default_image_1_0(self, load_config):
+        from doc_builder.constants import DOCKER_IMAGE_SETTINGS
```

Comment:
Can't this be imported at the beginning of the file?

Reply:
Yes,
I have pushed the changes